app/src/main/res/layout/quest_entrance_flat_range.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:divider="@drawable/space_8dp"
+    android:showDividers="middle">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:text="@string/quest_entrance_reference_flat_range_label"/>
+
+    <FrameLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        >
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            >
+
+            <me.grantland.widget.AutofitLayout
+                android:id="@+id/flatRangeStartInputContainer"
+                android:layout_width="128dp"
+                android:layout_height="wrap_content"
+                android:background="@drawable/background_white_corners"
+                android:layout_gravity="center"
+                android:layout_marginTop="4dp"
+                android:layout_marginBottom="4dp"
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="16dp"
+                >
+
+                <EditText
+                    android:id="@+id/flatRangeStartInput"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@drawable/background_rectangular_sign_blue"
+                    android:imeOptions="actionDone"
+                    android:textColorHint="#2fff"
+                    android:digits="0123456789.,- /IVX"
```

Comment:
and that does work? Or does there ned to be an "ABC" button to switch the keyboard from numeric to letters?

Reply:
Good point. It does not work.

![screen01](https://user-images.githubusercontent.com/899988/175777400-af0785d7-2bbf-460f-ba19-15b9dc2075b3.png)

Reply:
So there should be an ABC button that works like the one in the housenumber quest.